var/spack/repos/builtin/packages/py-cmake-format/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyCmakeFormat(PythonPackage):
+    """cmake-format project provides Quality Assurance (QA) tools for cmake.
+
+- cmake-annotate can generate pretty HTML from your listfiles
+- cmake-format   can format your listfiles nicely so that they don’t look
+                 like crap.
+- cmake-lint     can check your listfiles for problems
+- ctest-to       can parse a ctest output tree and translate it into a more
+                 structured format (either JSON or XML)."""
```

Comment:
I don't think this will format well, you can check `spack info py-cmake-format` to confirm. I would just remove everything except the first line of the description.

Reply:
Fixed in 834f97e.